target/linux/octeon/patches-5.4/150-octeon_e300_support.patch

```diff
@@ -0,0 +1,18 @@
+--- a/arch/mips/include/asm/octeon/cvmx-bootinfo.h
++++ b/arch/mips/include/asm/octeon/cvmx-bootinfo.h
+@@ -296,6 +296,7 @@
+ 	CVMX_BOARD_TYPE_CUST_PRIVATE_MIN = 20001,
+ 	CVMX_BOARD_TYPE_UBNT_E100 = 20002,
+ 	CVMX_BOARD_TYPE_ITUS_SHIELD = 20006,
++	CVMX_BOARD_TYPE_UBNT_E300 = 20300,
+ 	CVMX_BOARD_TYPE_KONTRON_S1901 = 21901,
+ 	CVMX_BOARD_TYPE_CUST_PRIVATE_MAX = 30000,
+ 
+@@ -397,6 +398,7 @@
+ 		ENUM_BRD_TYPE_CASE(CVMX_BOARD_TYPE_CUST_PRIVATE_MIN)
+ 		ENUM_BRD_TYPE_CASE(CVMX_BOARD_TYPE_UBNT_E100)
+ 		ENUM_BRD_TYPE_CASE(CVMX_BOARD_TYPE_ITUS_SHIELD)
++		ENUM_BRD_TYPE_CASE(CVMX_BOARD_TYPE_UBNT_E300)
```

Comment:
Any reason for that sorting? (Same above ...)

Reply:
kernel use sorting by board id.
that is the only reason.

Reply:
Ah okay, fine with me.